app/src/org/commcare/preferences/DeveloperPreferences.java

```diff
@@ -315,4 +317,8 @@ public static String formLoadPayloadStatus() {
     public static boolean isDetailTabSwipeActionEnabled() {
         return doesPropertyMatch(DETAIL_TAB_SWIPE_ACTION_ENABLED, CommCarePreferences.YES, CommCarePreferences.YES);
     }
+
+    public static boolean isOfflineUpdateEnabled() {
+        return doesPropertyMatch(OFFLINE_UPDATE_ENABLED, CommCarePreferences.YES, CommCarePreferences.YES);
```

Comment:
Is there a reason to flag this at all if we are going to enable it by default? It's hard for me to imagine someone getting a ton of value from disabling the flag.